fossology/fossology
# feat(spdx2): Ignore files with no info in SPDX reports

## Description

Implement ignore files with no info in SPDX reports (see #1161)


## How to test

1) Activate "ignore files with no info in SPDX" in info page and launch an export SPDX. 
2) Verify only files with info are in SPDX export.

closes #1161 

![refofNewReportInfoPage](https://user-images.githubusercontent.com/10155976/58707946-f7308100-83d3-11e9-8b44-5cd314fee344.PNG)
